codegen/templates/workflow.tmpl

```diff
@@ -30,6 +30,7 @@ import (
 	{{- end}}
 
 	module "{{$instance.PackageInfo.ModulePackagePath}}"
+	"go.uber.org/zap"
```

Comment:
nit: move it to https://github.com/uber/zanzibar/pull/374/files#diff-f66420b147eeb736922ebdd1ea765a69R20

Reply:
mm ... I was following this: https://github.com/golang/go/wiki/CodeReviewComments#imports

`go.uber.org` considered 3rd party package in this import